feature-libs/product-configurator/common/shared/testing/common-configurator-test-utils.service.ts

```diff
@@ -62,4 +62,27 @@ export class CommonConfiguratorTestUtilsService {
         htmlElement.innerHTML
     );
   }
+
+  /**
+   * Helper function for proving how many times the element comes in the DOM tree.
+   *
+   * @param {any} expect - Expectation for a spec.
+   * @param {Element} htmlElement - HTML element.
+   * @param {string} querySelector - Query selector
+   * @param {number} expectedNumber- expected number of elements
+   */
+  static expectNumberOfElements(
```

Comment:
Only now I discovered we bundle this testing util into the production code for no reason (it has been like that since 3.1). 

Created ticket https://github.com/SAP/spartacus/issues/12096

Reply:
Ok, thanks for creating that ticket